content.js

```diff
@@ -17,3 +16,9 @@ if (do_scrollbar){
     console.log('adding scroll bar')
     document.querySelector('html').style.overflowY = 'auto';
 }
+else {
+    console.log('doing extreme stuff')
+    if (!document.location.href.includes('172')){
+         window.location.replace(document.location.href + '?172')
+    }
+}
```

Comment:
The fix is simply redirecting from `<URL>` to `<URL>?172`. It's ugly, but it doesn't look like a refresh (try it yourself!)